plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1194,15 +1270,44 @@ function VocabBuilder:addToMainMenu(menu_items)
     }
 end
 
+function VocabBuilder:getWordContext(pos)
+    if not pos then return end
+    local pos_start = pos[1]
```

Comment:
I think this generic function could go into readerhighlight.lua as `ReaderHighlight:getSelectedWordContext(nb_words)` - it might be useful for other bits/plugins.
You would just call it here as `self.ui.highlight.getSelectedWordContext(15)`.
(And you would check in it that `if not self.ui.rolling then return nil end`)